kokoro/ubuntu/new_release.sh

```diff
@@ -23,24 +24,19 @@ test -n "${ANALYTICS_TRACKING_ID}"
 
 cd git/google-cloud-eclipse
 
-# Drop the `.qualifier` in `metadata.product` and use the exact release
-# version. (For example, `version="1.3.0.qualifier"` to `version="1.3.0"`.)
-# version.
-#
-# https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2363#issuecomment-327844378
-# https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/2211
-readonly CT4E_DISPLAY_VERSION=$( \
-  xmlstarlet sel -t -v '/product/@version' gcp-repo/metadata.product )
-readonly NO_QUALIFIER_VERSION="${CT4E_DISPLAY_VERSION%.qualifier}"
-xmlstarlet ed --inplace -u '/product/@version' -v "${NO_QUALIFIER_VERSION}" \
-  gcp-repo/metadata.product
+VERSION_SUFFIX_PROPERTY=
+if [ "${PRODUCT_VERSION_SUFFIX}" ]; then
+  VERSION_SUFFIX_PROPERTY="'${PRODUCT_VERSION_SUFFIX}'"
```

Comment:
Worth a comment here that the single-quotes are necessary so its interpretation as a date format in the `<format>` expression.